generalhw/hpt630v1/README.md

```diff
@@ -0,0 +1,50 @@
+![Setup diagram](../openqa-qubesos-setup.png)
+
+For details about how this setup works, see
+[../adding-vnc-setup.md](../adding-vnc-setup.md).
+
+### Start the job
+
+While logged into openQA server (unless you've installed and configured
+`openqa-cli` locally):
+
+Perform installation followed by AEM testing:
+
+```
+openqa-cli api -X POST isos DISTRI=qubesos VERSION=4.2 ARCH=x86_64 BUILD=4.2.0 FLAVOR=install-iso-optiplex
+```
+
+ISO name is generated as `ISO=Qubes-R%BUILD%-%ARCH%.iso`, you should also create
+a symlink to it in `hdd` directory.  Only Fedora template is going to be
+installed to make setup faster.
+
+Output like
+
+```
+{"count":4,"failed":[],"ids":[448,449,450,451],"scheduled_product_id":59}
+```
+
+means the jobs (`4` in this case) were scheduled successfully.
+
+Use "Dependencies" tab to see jobs which are part of the same run.
+
+#### Parameters
+
+These settings can be added to `openqa-cli` posting command to specify which
+packages to use.  Current default uses
+<https://dl.3mdeb.com/open-source-firmware/QubesOS/trenchboot_aem_v0.3/>.
```

Comment:
Update according to whatever will be used in template. The same may be needed for other platforms, so maybe this should be made generic and point to template so we won't have to bump it with each new version.